libnd4j/tests_cpu/libnd4j_tests/CMakeLists.txt

```diff
@@ -21,10 +21,10 @@ endif()
 # OPTIONAL ONE-DNN
 if ("${BUILD_ONEDNN}")
     # Download and unpack mkl-dnn at configure time
```

Comment:
Should this be mkldnn -> onednn? I remember renamign this

Reply:
it was missing if I remember correctly